.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: ruby
 script:
-  - bundle exec rubocop -D
-  - bundle exec rake spec:all
+  - bundle exec rake
```

Comment:
I think travis runs `rake` as default but probably without `bundle exec` or maybe, not sure - also it's better to be explicit. So this is good. I don't know what I'm writing ;P